Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,13 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
+#:
+
+# shellcheck disable=SC2155
+homebrew-formulae() {
+  local formulae="$(
+    find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name spec -o -name lib -o -name cmd -o -name .github \) -prune -false -o -name '*\.rb' |
+    sed -E -e 's/\.rb//' -e 's .*/Taps/(.*)/homebrew- \1/ ' -e 's /Formula/ / ')"
```

Comment:
You're right, I've updated it.  However, this won't work as is, because the '|' symbol is reserved for boolean or, so I changed it to underscore (unless anyone has a better idea).